frontend/src/pages/RecoverPage.jsx

```
@@ -115,165 +158,92 @@ const RecoverPage = () => {
+    />
+  </svg>
+);
+
+const ConfirmPassIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    fill="black"
+    stroke="white"
+    strokeWidth="1"
+  >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)